spark/v3.4/build.gradle

```diff
@@ -135,6 +135,7 @@ project(":iceberg-spark:iceberg-spark-extensions-${sparkMajorVersion}_${scalaVer
 
   dependencies {
     implementation("org.scala-lang.modules:scala-collection-compat_${scalaVersion}")
+    implementation "org.roaringbitmap:RoaringBitmap"
```

Comment:
We already depend on `RoaringBitmap` for vectorized reads with deletes. It is already relocated.